src/components/organisms/SearchPane/RecentSearch.tsx

```diff
@@ -0,0 +1,23 @@
+import electronStore from '@utils/electronStore';
+
+import * as S from './styled';
+
+type RecentSearchProps = {
+  handleClick: (query: string) => void;
+};
+
+const RecentSearch = ({handleClick}: RecentSearchProps) => {
+  const recentSearch: string[] = electronStore.get('appConfig.recentSearch')?.reverse() || [];
```

Comment:
Should we maybe add a global variable for this, maybe it might be useful in other components?

Reply:
good point, moved!